### DISCUSSION TITLE
How and where to run this "ytmusicapi oauth"

### DISCUSSION THREAD

**Discussion by shivardev** [sigma67/ytmusicapi#411] [Q&A]
I ran pip install ytmusicapi as mentioned in the docs.
right after that i have to call "ytmusicapi oauth" but I'm getting this
'ytmusicapi' is not recognized as an internal or external command,
operable program or batch file.
.
.
what should i add to my system path to fix this?
Win 11.
I have searched for a similar question and found one as well but it didn't help me

**Comment by sigma67** [comment]
I doubt you searched properly: https://github.com/sigma67/ytmusicapi/discussions/385
